Words added to wordlist.txt to tackle the spellcheck errors thrown (#73)
kubeslice/docs: versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/managing-namespaces.mdx

# Managing Namespaces

## Introduction

> This topic describes the steps to on-board namespaces to a slice. A namespace in Kubernetes 

Reviewer: ```suggestion
This topic describes the steps to onboard namespaces to a slice. A namespace in Kubernetes 
```

is a logical separation of resources in a cluster. Kubernetes resources such as pods and 
services are associated with a namespace and are guaranteed to be uniquely identifiable 
within it. 

> Namespaces that are created to run application deployments can be on-boarded on a slice to 

Reviewer: ```suggestion
Namespaces that are created to run application deployments can be onboarded on a slice to 
```

form a micro-network segment. After a namespace is bound to a slice, all the pods that get 
scheduled in the namespace would get connected to the slice. 

> ## on-boarding Namespaces

Reviewer: ```suggestion
## Onboarding Namespaces
```

> To on-board namespaces, you must add them as part of applicationNamespaces in the [slice configuration](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/creating-a-slice.mdx#creating-the-slice-yaml-file) YAML file.

Reviewer: ```suggestion
To onboard namespaces, you must add them as part of applicationNamespaces in the [slice configuration](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/creating-a-slice.mdx#creating-the-slice-yaml-file) YAML file.
```

You can add namespaces in the following ways in the slice configuration YAML file:

- Add namespaces for each worker cluster.
- Add a wildcard * (asterisk) to add all namespaces in the worker clusters.

:::info

> Ensure that the namespace that you want to on-board exists on the worker cluster.

Reviewer: ```suggestion
Ensure that the namespace that you want to onboard exists on the worker cluster.
```

```
:::info
Adding the asterisk (*) enables the namespace sameness, which means that the namespace 

> is on-boarded on all the worker clusters of that slice.

Reviewer: ```suggestion
is onboarded on all the worker clusters of that slice.
```

:::

To know more on parameters, see [namespace isolation profile parameters](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/configuration-parameters.mdx#namespace-isolation-profile-parameters).
This configuration ensures that all the application deployments from that namespace are 

> on-boarded automatically on to the slice.

Reviewer: ```suggestion
onboarded automatically on to the slice.
```

> ## off-boarding Namespaces

Reviewer: ```suggestion
## Offboarding Namespaces
```

> You can off-board namespaces from the slice and prevent the applications on the slice 

Reviewer: ```suggestion
You can offboard namespaces from the slice and prevent the applications on the slice 
```

> from accessing them. To know more, see [off-boarding namespaces](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/uninstalling-kubeslice/off-boarding-namespaces.mdx).

Reviewer: ```suggestion
from accessing them. To know more, see [offboarding namespaces](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/uninstalling-kubeslice/offboarding-namespaces.mdx).
```
